Finish development

Evanwalsh95 opened the pull request:
Finish development of site
Add styles to header files to match design given
Add proper spacing to transparent header meta file to fix stylelint warning
Add styles to home section to match design given
Create variables for blue, red and update black color selection
Add correct font-family to H2 styling

Commit:
Finish development

Finish development of site
Add styles to header files to match design given
Add proper spacing to transparent header meta file to fix stylelint warning
Add styles to home section to match design given
Create variables for blue, red and update black color selection
Add correct font-family to H2 styling

pwnrod commented on assets/scss/pages/_home.scss:
```diff
@@ -7,5 +7,134 @@ body#top.home {
+                }
+
+                @include media("<932px") {
+                    width: 100%;
+                }
+            }
+
+            .av-flex-placeholder {
+                width: 7.5%;
+            }
+
+            .lg-address-section {
+                @include media(">932px") {
+                    border-left: .2rem solid $white-color;
+                    margin-left: 0;
+                    padding-left: 11rem;
+                }
+
+                @include media("<932px") {
+                    border-top: .2rem solid $white-color;
+                    padding-top: 3.5rem;
+                    width: 100%;
+                    margin-left: 0;
+                }
+
+                h3,
+                p,
+                a,
+                strong {
+                    color: $white-color;
+                    font-family: "interstate", serif;
+                }
+
+                p,
+                a {
+                    font-weight: 400;
+                }
+            }
+        }
+    }
```
I would extract this chunk out into a partial, maybe in a /partials/home-partials directory. The styling for this one section is larger than the whole rest of the file, so extracting it out will make both files easier to work with.

pwnrod commented on assets/scss/partials/header-partials/_header-main.scss:
```diff
@@ -4,22 +4,67 @@ it simply bootstraps the navigation menu. This helps to eliminate some of the de
                     display: flex;
                 }
+
+                &::after {
+                    content: "Machinists";
+                    position: initial;
+                    height: .6rem;
+                    margin-left: 1.5rem;
+                    font-size: 1.8rem;
+                    color: $white-color;
+                    font-family: "interstate", serif;
+                    text-transform: uppercase;
+                }
```
This should go above the `a` style block, since these styles are technically still for `.logo`

pwnrod commented on assets/scss/partials/header-partials/_header-main.scss:
```diff
@@ -4,22 +4,67 @@ it simply bootstraps the navigation menu. This helps to eliminate some of the de
+                }
+            }
+
+            @include media("<tablet") {
+                align-items: center;
+                display: flex;
+                justify-content: center;
+            }
+
+            @include media('<=320px') {
+                height: 11rem;
+
+                .logo {
+                    max-width: 18rem;
+                }
```
These look like media queries for the `.inner-container` selector, so they should be above the `.logo` style block.

Commit:
Pull request changes

Break out contact section into separate partial for ease of use
Move media queries up to be under selector it applied to
Move after statement to after that a tag

pwnrod commented on assets/scss/partials/lg-classes/_lg-contact-section.scss:
```diff
@@ -0,0 +1,82 @@
+                font-size: 1.8rem;
+            }
+
+            @include media("<932px") {
+                width: 100%;
+            }
```
This media query should be moved to the top of the style block

Commit:
Pull request changes

Move media query to proper position
